src/components/MbtiTest/startTest.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styles from './startTest.module.scss';
+import Button from '../styled-components/Button';
+
+const StartTest = () => {
+  const [userName, setUserName] = useState('');
+  const navigate = useNavigate();
+  const onSubmit = (e) => {
+    e.preventDefault();
+    navigate('/mbtiTest', { state: { name: userName } });
+  };
+  const onChange = (e) => {
+    setUserName(e.currentTarget.value);
+  };
```

Comment:
```suggestion
  const onChange = (e: SyntheticEvent<HTMLInputElement>) => {
    setUserName(e.currentTarget.value);
  };
```
타입을 명시하면 더 좋을 거 같아요!